### PR TITLE
[fuchsia] Filter out example fuzzers.

### PR DESCRIPTION
**Pull request by flowerhack**
Fuchsia has a set of example fuzzers that are useful for testing, but
which we only want selected if it's explicitly set via FUZZ_TARGET.

Otherwise, we only want "real" fuzzers to be picked during standard runs
(e.g. on Clusterfuzz bots).

### CODE REVIEW

File: src/python/platforms/fuchsia/util/fuzzer.py [hunk 1]
```diff
@@ -69,7 +69,18 @@ def _matches_sanitizer(cls, tgt, sanitizer):
...
 
   @classmethod
-  def filter(cls, fuzzers, name, sanitizer=None):
+  def _is_example(cls, pkg, tgt):
+    """ Returns whether or not a given pkg/tgt pair is an example fuzzer.
+    (Helper function to prevent us from wasting cycles on example fuzzers in
+    production). """
+    # We want to remove the noop-fuzzer regardless of the extension.
+    tgt = os.path.splitext(tgt)[0]
+    if pkg == 'example_fuzzers' or tgt == 'noop-fuzzer':
```
Comment (oliverchang) [inline src/python/platforms/fuchsia/util/fuzzer.py]: nit: might be more concise as 

return (pkg == .. or tgt == )

File: src/python/build_management/build_manager.py [hunk 2]
```diff
@@ -746,7 +739,7 @@ def _get_fuzz_targets_from_dir(self, build_dir):
         environment.get_value('JOB_NAME')).lower()
     return [
         str(target[0] + '/' + target[1])
-        for target in Fuzzer.filter(host.fuzzers, '', sanitizer)
+        for target in Fuzzer.filter(host.fuzzers, '', sanitizer, False)
```
Comment (oliverchang) [inline src/python/build_management/build_manager.py]: nit: use example_fuzzers=False for better readability.

**Comment by oliverchang** [comment]
Please do a reformat :) The line you just changed is now too long.

**Comment by inferno-chromium** [comment]
I will do a refomat later, committing for now. CI would fail this CL is not rebased to https://github.com/google/clusterfuzz/pull/1203

**Pull request merged**